platform/local/cluster.go

```diff
@@ -102,7 +102,7 @@ func (lc *LocalCluster) GetDiscoveryURL(size int) (string, error) {
 	client := &http.Client{Transport: tr}
 
 	body := strings.NewReader(url.Values{"value": {strconv.Itoa(size)}}.Encode())
-	req, err := http.NewRequest("PUT", URL+"_config/size", body)
+	req, err := http.NewRequest("PUT", URL+"/_config/size", body)
```

Comment:
Uppercase variable names are generally reserved for exported variables, not local function variables, please call it something like `base` or something.